marklogic-client-api/src/main/java/com/marklogic/client/type/PlanDocColsIdentifier.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.client.type;
+
+public interface PlanDocColsIdentifier extends PlanDocColsIdentifierSeq {
```

Comment:
It will be necessary to add fluent setters for the fields of a doc cols identifier as part of the manual updates to enable update operations in a future pull request.